app/knowledge_panels.py

```diff
@@ -39,3 +38,50 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def data_quality_kp(
+    facet,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    """
+    Get data corresponding to differnet facet
+    """
+    path = ""
+    description = ""
+    if facet == "country":
+        country = value
+        country_code = country_to_ISO_code(value=value)
+        url = f"https://{country_code}-en.openfoodfacts.org"
+        path = ""
+        facet = value = None
+    if country is not None:
+        country_code = country_to_ISO_code(value=country)
+        url = f"https://{country_code}-en.openfoodfacts.org"
+        path = ""
+        description += country
+    if country is None:
+        url = "https://world.openfoodfacts.org/"
+    if facet is not None:
+        path += facet
+        description += f"{facet}"
+    if value is not None:
+        path += f"/{value}"
+        description += f" {value}"
+    description = f"Data-quality issues related to {description}"
+    (expected_html, source_url) = dataQuality(url=url, path=path)
```

Comment:
it's strange to use `expected_html` as a name here ("expected" is in tests).
Maybe `quality_html` or just `html` ?